cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -15,6 +15,17 @@ describe('Top toolbar tests.', function() {
 		helper.afterAll(testFileName);
 	});
 
+	it('Apply style combobox', function() {
+		cy.get('#tb_editbar_item_styles')
+			.click();
+	
+		cy.contains('.select2-results__option','Index')
+			 .click();
+	
+		cy.get('#copy-paste-container p')
+			.should('have.attr', 'style', 'margin-bottom: 0in; line-height: 100%');
```

Comment:
I'm not really like this check this way. It would be good to be more specific about what is changed by applying a font style. margin-bottom and line-height are the same when I open the test document. Maybe you should use a different style that has a more visible change under '#copy-paste-container' element. See a similar mobile test: https://github.com/CollaboraOnline/online/blob/master/cypress_test/integration_tests/mobile/writer/apply_font_spec.js#L231. There I used 'Title' style which changes the font name (e.g. Liberation Sans, sans-serif) and the font size too (e.g. 28pt).